packages/react-native-client/.eslintrc.js

```diff
@@ -0,0 +1 @@
+module.exports = require('../../.eslintrc.js');
```

Comment:
Curious: why this change?

Reply:
`expo prepare` generates this file, but with a wrong path. Adding it manually fixes the issue.